ci/prow/boskos/config.yaml

```diff
@@ -104,7 +104,7 @@ metadata:
     app: boskos-janitor
   namespace: test-pods
 spec:
-  replicas: 12  # 12 distributed janitor instances
+  replicas: 40  # This number distributed janitor instances. Should keep around the same number as number of projects.
```

Comment:
I believe 40 is high number. Do we know how long the resource contention happened for? We need the higher number of boskos projects as they run for ~30 min in presubmits and >1 hr on periodic (perf tests run for 2 hrs). But, the janitors dont run as long, and I dont think we need one per project. Maybe try with half, 20?

Reply:
Set to 20.